### PR TITLE
refresh button for statement browser

### PR DESCRIPTION
**Pull request by milt**
Allows live-refreshing of browser

### CODE REVIEW

File: src/com/yetanalytics/lrs_admin_ui/views/credentials/api_key.cljs
Comment (milt) [inline src/com/yetanalytics/lrs_admin_ui/views/credentials/api_key.cljs:1]: I have no idea how this file I never opened got roped in to this haha

File: src/com/yetanalytics/lrs_admin_ui/views/browser.cljs [hunk 1]
```diff
@@ -151,6 +151,15 @@
...
                  :width "30px"}]])]]]))
 
+(defn refresh-button []
+  (when @(subscribe [:browser/get-credential])
+    [:a.icon-refresh
+     {:href "!#"
+      :on-click (fn [e]
+                  (fns/ps-event e)
+                  (dispatch [:browser/refresh]))}
+     "Refresh"]))
```
Comment (cliffcaseyyet) [inline src/com/yetanalytics/lrs_admin_ui/views/browser.cljs:161]: Prob want to use use localization / override system since this is visible text.

Comment (milt) [reply src/com/yetanalytics/lrs_admin_ui/views/browser.cljs:161]: oh right

Comment (milt) [reply src/com/yetanalytics/lrs_admin_ui/views/browser.cljs]: done https://github.com/yetanalytics/lrs-admin-ui/pull/79/commits/a294481677394430a2bcc2a094d341b723456a59

**Review by cliffcaseyyet — APPROVED** [review]

**Pull request merged**